src/python.cc

```diff
@@ -745,8 +745,15 @@ ModelInstanceState::SetupChildProcess()
   int64_t shm_default_size =
       model_state->StateForBackend()->shm_default_byte_size;
 
-  shm_pool_ = std::make_unique<SharedMemory>(
-      shm_region_name, shm_default_size, shm_growth_size, true /* truncate */);
+  try {
+    shm_pool_ = std::make_unique<SharedMemory>(
+        shm_region_name, shm_default_size, shm_growth_size,
+        true /* truncate */);
+  }
+  catch (const PythonBackendException& pb_exception) {
+    return TRITONSERVER_ErrorNew(
+        TRITONSERVER_ERROR_INTERNAL, pb_exception.what());
```

Comment:
How about:
Unable to create shared memory region + "shm_region_name" + ": " + pb_exception.what()

Reply:
@CoderHam  all the returned exceptions have `shm_region_name` in them so I think it would be redundant.

Reply:
Oh yes, I missed that you were mentioning it there.